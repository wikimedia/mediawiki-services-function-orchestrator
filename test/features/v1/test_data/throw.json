{
	"Z1K1": "Z7",
	"Z7K1": "Z820",
	"Z820K1": {
		"Z1K1": "Z5",
		"Z5K1": {
			"Z1K1": "Z50",
			"Z50K1": []
		},
		"Z5K2": "This is a real error; you're in trouble"
	}
}
