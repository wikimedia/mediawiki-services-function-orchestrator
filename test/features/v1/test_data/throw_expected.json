{
	"Z1K1": "Z5",
	"Z5K1": {
		"Z1K1": "Z50",
		"Z50K1": [
			"Z1"
		]
	},
	"Z5K2": "This is a real error; you're in trouble"
}
